<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serviceTech</class>
 <widget class="QDialog" name="serviceTech">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-5</x>
     <y>-5</y>
     <width>801</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="cinemApp.qrc">:/new/prefix1/cinemApp.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>69</y>
     <width>801</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Gestion des salles</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>371</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajout / modification</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Num° salle</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Capacité</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>état</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="numsalle">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="capacitesalle">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifS">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>110</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajoutsalle">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="etatsalle">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>371</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Suppression</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Num° salle</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_suppSalle">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_suppSalle">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>419</x>
       <y>29</y>
       <width>361</width>
       <height>301</height>
      </rect>
     </property>
     <property name="title">
      <string>Liste des salles</string>
     </property>
     <widget class="QTableView" name="tableView_salle">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>321</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pdf">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Exopter en PDF</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Gestion employés</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>351</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajout / modification</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Num° tel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_IDE">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nomE">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prenomE">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_numE">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifEmp">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajoutE">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>160</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="arduinoh">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>120</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>rfid</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>351</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Suppression</string>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_supprimerEmploye">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimerEmploye">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>389</x>
       <y>29</y>
       <width>381</width>
       <height>321</height>
      </rect>
     </property>
     <property name="title">
      <string>Liste des employés</string>
     </property>
     <widget class="QTableView" name="tableView_employe">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>331</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Tri">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="expoter_pdf">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Expoter en EXCEL</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Statistique </string>
    </attribute>
    <widget class="QCustomPlot" name="plot" native="true">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>9</y>
       <width>751</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="cinemApp.qrc"/>
 </resources>
 <connections/>
</ui>
